packages/cascara/src/atoms/TimePicker/TimePicker.js

```diff
@@ -15,19 +15,27 @@ const propTypes = {
 };
 
 const TimePicker = ({ format, lang, onChange, ...rest }) => {
+  const [isOpen, setOpen] = useState(false);
   const handleOnChange = useCallback(
     (time, timeString) => {
+      setOpen(false);
```

Comment:
is this the best route (explicit false) instead of just flipping the value of isOpen?
What actions mark this value as true?

State management for booleans might get tricky, depending on this logic the easier thing for toggle actions is just to flip the value.

Reply:
We don't need it. The function below will make the toggle functionality. Thank you for pointing this out.